Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -47,7 +47,7 @@ JitArm64::~JitArm64() = default;
 
 void JitArm64::Init()
 {
-  RefreshConfig();
+  RefreshConfig(InitFastmemArena::Yes);
```

Comment:
Shouldn't the existing `jo.fastmem_arena = m_fastmem_enabled && memory.InitFastmemArena();` in this function be removed too?

Reply:
Whoops, yes.